build/install_base.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# hadolint ignore=SC2086,DL3015,DL3008,DL3013,SC2015
```

Comment:
I'm not sure what to do with this. I can change to shellcheck but it doesn't support the DL suppressions. I should probably figure out if they're even still needed but thought it would be worth your input first.

Reply:
I'm not getting any warnings, so I'm going to assume that these were all corrected during my refactoring.